testdata/dnn/onnx/data/README.md

```diff
@@ -0,0 +1,22 @@
+### OpenCV: Open Source Computer Vision Library
+
+This repository contains extra data for the OpenCV library.
```

Comment:
Why do we have this file here in such place?

Reply:
I'm so sorry, it was generated by a wrong operation.